distribution/src/main/resources/bin/plugin.bat

```diff
@@ -9,9 +9,46 @@ for %%I in ("%SCRIPT_DIR%..") do set ES_HOME=%%~dpfI
 
 TITLE Elasticsearch Plugin Manager ${project.version}
 
+SET properties=
+SET args=
+
+:loop
+SET "current=%~1"
+SHIFT
+IF "x!current!" == "x" GOTO breakloop
+
+IF "!current:~0,2%!" == "-D" (
```

Comment:
That does not work with java. We should just stick with the way the jvm takes sysprops...
```
[14:04:26][~/Code/test]$ java -cp . Test
null
[14:04:50][~/Code/test]$ java -cp . -Dfoo bar Test
Error: Could not find or load main class bar
[14:05:03][~/Code/test]$ java -cp . -Dfoo=bar Test
bar
```